test/integration/src/daml.yaml

```diff
@@ -1,4 +1,4 @@
-sdk-version: 0.12.25
+sdk-version: 0.13.30
```

Comment:
```suggestion
sdk-version: 0.12.25
```
I understand it's old, but let's keep the SDK versions in sync with the rest of the library.